### ISSUE TITLE
[nfr] Support for pcap@3

### ISSUE THREAD

**Issue by ruudboon** [njh/node-red-contrib-pcap#4]
Currently the default install fails since pcap 3 is been used. Could you add support for this?

**Comment by njh** [comment]
Ooh, I kind of gave up on this because I was finding pcap so unstable and new versions / fixes weren't getting released.

Fingers crossed v3 makes everything better 🤞

**Comment by ruudboon** [comment]
@njh Trying to get it working now. Want me to drop a pull on this? Looks like it's broken in the first place since it's trying to include pcap2 instead of pcap

**Comment by njh** [comment]
Yeah, it was loading pcap2 (https://github.com/andygreenegrass/node-pcap) because of the lack of releases of the official version.

A Pull Request would be very welcome.
